docs/resources/vault_radar_source_github_enterprise.md

```diff
@@ -0,0 +1,45 @@
+---
+page_title: "hcp_vault_radar_source_github_enterprise Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  This Vault Radar resource manages HCP Cloud scans of GitHub Enterprise sources.
+---
+
+# hcp_vault_radar_source_github_enterprise (Resource)
+
+-> **Note:** HCP Vault Radar Terraform resources are in preview.
+
+This Vault Radar resource manages HCP Cloud scans of GitHub Enterprise sources.
+
```

Comment:
nit: While we call it `github_enterprise` on the Radar side, can we specify somewhere that this is actually for GitHub Enterprise Server (I think the terminology is that "Enterprise" is just paid, so it could be GitHub Enterprise Cloud as well, even though we call that just `github_cloud`) 

The portal has added the differentiation as so as well

<img width="406" alt="Screenshot 2024-09-16 at 4 23 09 PM" src="https://github.com/user-attachments/assets/32e36fec-6f4b-4eb6-904e-29477bc7f5d1">

Reply:
I can certainly update the docs, not sure if I want to change the resource name as its pretty long as is.

Reply:
Update docs, didn't change the name of the resource.